src/components/Shop/ShopItems.tsx

```diff
@@ -0,0 +1,60 @@
+import { MdOutlineAddShoppingCart } from 'react-icons/md';
+import { Link } from 'react-router-dom';
+import { Image } from '../../types';
+import Pagination from '../helpers/Pagination';
+import useDocsSnapshot from '../helpers/useDocsSnapshot';
+import usePaginateData from '../helpers/usePaginateData';
+
+interface ShopItemsProps {
+  addToCart: (item: Image) => void;
+}
+
+function ShopItems({ addToCart }: ShopItemsProps) {
+  const { data: shopItems } = useDocsSnapshot<Image>(`gallery`, [], {
+    whereArg: ['price', '>', 0],
+    orderByArg: ['price', 'desc'],
+    secOrderByArg: ['createdAt', 'desc'],
+  });
+
+  const paginatedData = usePaginateData<Image>(shopItems);
+
+  return (
+    <div className="container">
+      <div className="shop-full">
+        <h2 className="headerTextStroke">Shop</h2>
+        <h3 className="headerwTextStroke">Shop</h3>
```

Comment:
Dwa headery są potrzebne dla tego? Na pewno nie da się zrobić inaczej?

Reply:
Mimo że zaznaczyłeś jako rozwiązaną wciąż jestem ciekaw po co ci dwa nagłówki obok siebie. Bo jeśli to kwestia styli, to to można rozwiązać inaczej. Tak to masz dwa razy shop tuż obok siebie. Osoby korzystające z czytników byłyby bardzo zdziwione tym faktem.

Reply:
Własnie myslałem ze będę miał po jednym takim zestawie na strone kazda wiec dla czytelnosci dalem w headerach a style i tak są do klas przypisanych